templates/iiif-images-compose.yml

```diff
@@ -5,8 +5,8 @@ services:
     environment:
       AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
       AWS_SECRET_KEY: ${AWS_SECRET_ACCESS_KEY}
-      S3CACHE_BUCKET_NAME: ${S3_SOURCE_BUCKET_NAME}/cantaloupe/cache #NO TRAILING SLASH!
-      S3_SOURCE_BUCKET_NAME:
+      S3CACHE_BUCKET_NAME: yul-dc-dev-image-samples/cantaloupe/cache #NO TRAILING SLASH!
```

Comment:
could we leave ${S3_SOURCE_BUCKET_NAME} in there, instead of repeating the bucket in 2 places?

Reply:
the point of this ticket is to get the local development and ecs versions pointing at different buckets.  I'm going to test this out.

Reply:
Testing in local development confirms that adding oids locally will generate ptiffs in the correct bucket and it is also pulling manifests from the correct bucket as well.  Though to see images from s3 one must export the following variables into the shell running one's containers before running cam up:
```
export AWS_PROFILE=yale AWS_DEFAULT_REGION=us-east-1 VPN=false MANAGEMENT_VERSION=v2.21.2 BLACKLIGHT_VERSION=v1.33.0 IIIF_MANIFEST_VERSION=v3.0.1 IIIF_IMAGE_VERSION=v1.0.2

```
In testing it was found that the latest release of the IIF Manifest would not serve the manifests so this is why the versions are explicitly listed here so that this functionality can be reproduced.